cob_hardware_config/common/raw3_base_driver.yaml

```diff
@@ -0,0 +1,79 @@
+sync:
+  interval_ms: 20
+  overflow: 0
+heartbeat:
+  rate: 20
+  msg: "77f#05"
+
+defaults:
+  eds_pkg: cob_hardware_config
+  eds_file: "common/ElmoSimplIQ.dcf"
+  dcf_overlay:
+    "606A": "1" # read velocity from velocity sensor
+    "607F": "240000" # max profile velocity
+    "6083": "1000000" # profile acceleration
+    "6084": "1000000" # profile deceleration
+    "6085": "1000000" # quick stop deceleration
+  motor_allocator: cob_elmo_homing::ElmoMotor402::Allocator
+  motor_layer:
+    monitor_mode: false
+
+drive_wheel: &drive_wheel
+  dcf_overlay: # "ObjectID" : "ParameterValue" (both as strings)
+  # "6098": "35" # homing method
```

Comment:
This is just the copy of the raw3-5 config.  For the legacy base homing is not used. It is commented out anyway.